src/ontology/mondo-ingest.Makefile

```diff
@@ -398,8 +421,9 @@ signature_reports: $(ALL_MIRROR_SIGNTAURE_REPORTS) $(ALL_COMPONENT_SIGNTAURE_REP
 # - Doeesn't include: broad mappings
 # - Comes from make tmp/mondo.owl
 
-tmp/mondo.sssom.ttl: tmp/mondo.sssom.tsv
-	sssom convert $< -O rdf -o $@
+tmp/mondo.sssom.ttl:
+	$(MAKE) up-to-date-mondo.sssom.tsv
```

Comment:
## Usage of new goals
The prereqs `tmp/mondo.owl` and `tmp/mondo.sssom.tsv` have been moved from the goal prereqs position into the body of the goal and called using `$(MAKE)`. This ensures that the goal isn't run all of the time, even if nothing has changed (as that is what happens if you have a `.PHONY` goal as a prereq).

Also fixed any references that would have been broken due to positional syntax, e.g. `$<`.